plugins/codex-usage-stick/scripts/agent_hub_hook_tee.py

```
@@ -0,0 +1,122 @@
+    if bridge_argv and bridge_argv[0] == "--":
+        bridge_argv = bridge_argv[1:]
+    if not bridge_argv and not args.hub_only:
+        print("agent_hub_hook_tee: missing Island bridge command after --", file=sys.stderr)
+        return 2
+
+    raw = sys.stdin.buffer.read()
+    try:
+        hook = json.loads(raw.decode("utf-8"))
+        if not isinstance(hook, dict):
+            hook = {"raw": True}
+    except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)